### DISCUSSION TITLE
又发现一个经常出现的问题，麻烦大佬帮忙看看。

### DISCUSSION THREAD

**Discussion by Makerlong** [kkkgo/PaoPaoGateWay#41] [问和答] [answered]
就是目前多数时间全部正常使用，但是总会出现如图所示状况。要再重启gateway一下，paodns关闭了CNFALL（我是动态公网NAT1所以就关了，不知道合不合理麻烦大佬给看看）。目前UDP已经关闭，编译的嗅探iso。麻烦作者大佬帮忙瞅瞅那里问题嗯。
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/129925998/2351c2e7-6386-442a-a776-681066ef901e)
网页可以打开，但是不能播放。节点是IEPL、普通节点测试也是这样。
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/129925998/bd2cbcf2-0780-406c-a232-657ad9e1506a)

**Comment by kkkgo** [comment, marked as answer]
我翻了下你的配置：https://github.com/kkkgo/PaoPaoGateWay/discussions/39 
你把静态路由配置成/24了，意味着超过255个域名之后就无法访问。

**Reply by kkkgo** [reply]
CNFALL可以优化首次访问域名的解析速度，大多数网络条件下推荐开启。

**Reply by Makerlong** [reply]
好的，我修改试一试
